vm.go

```diff
@@ -21,9 +40,9 @@ type VirtualMachineSpec struct {
 	system    *hcs.System
```

Comment:
Is there a reason this is not merged?

Reply:
This PR should be merged.  Pending on aagiesek to merge the PR.

Reply:
@nagiesek  can you merge this please